### DISCUSSION TITLE
Disable prometheus UI

### DISCUSSION THREAD

**Discussion by rosh0450** [prometheus/prometheus#13748] [Q&A]
How to disable prometheus UI ??

**Comment by danizavtz** [comment]
I was looking for a feature flag to disable web interface too, I just would like to access database and query data straight to my web interface application. (like grafana or skywalking). I also do not want to see a web interface in default port for service. 

I was wondering if you can't disable interface, if it is possible to make available a compiled version without the web interface. Something like tsdb only. (or a not so often available version like a lts version without interface)

Are there any discussions about that? Are there any resources about it ?

Thanks.
